modules/apps/upload/upload-api/src/main/java/com/liferay/upload/BaseUploadHandler.java

```diff
@@ -12,7 +12,7 @@
  * details.
  */
 
-package com.liferay.portal.kernel.upload;
+package com.liferay.upload;
```

Comment:
Hi @dantewang @yuhai,
We need to remove this base class instead of moving it to another package cause it is no longer needed by the new indexer pattern.

So to remove the usages of it, we need to modify the test logic to follow the new indexer pattern.

Tina.